### ISSUE TITLE
Alt text should be descriptive

### ISSUE THREAD

**Issue by discospiff** [shaherna/IS7012#5]
For accessibility: in image tags, consider using an alt attribute, with a good yet concise description of the image.

https://github.com/shaherna/IS7012/blob/dfc04a43c160c79e0003fe7d9d2f276cebb5c019/naina_shaheryar-about-me.html#L49
